### ISSUE TITLE
Some thoughts about ~nocolor ~notexture ~nohide  e.t.c.

### ISSUE THREAD

**Issue by ghost** [OpenCollar/opencollar#698]
may be better "invert" description?
write ~сolor~texture~hide (e.t.c.)   only in those prims where it is necessary,
instead of ~nocolor~notexture~noshiny~noglow~nohide in all others?

**Comment by WendyStarfall** [comment]
I've thought the same very often and it just itches you in the fingers to change that, doesn't it?

In previous API versions this would have been problematic with updates, so it never was actually done.

Anyone who might be wondering what this is about:

Right now we only have to give an element a ~no... tag if this element should be customized in i.e. color but no texture. In that sense, as long as an element is not "named" at all, we don't have to worry about tags in descriptions. However, if looking at the creative process itself, the way to handle this is non-ideal. I rather would decide "And this piece, I want to be able to color" so add a ~color tag to it. The way it currently is handled, works backwards and the process would have to be "And this piece, I don't want to be able to texture but I want to be able to color it" ~notexture and little Miss collar-maker gets a headache.

tl;dr - now is a great chance to finally make this change

**Comment by RomkaSwallowtail** [comment]
if you're interested, I have a ready script:
https://github.com/RomkaSwallowtail/opencollar/blob/master/src/collar/oc_disgraced_themes.lsl

**Comment by RomkaSwallowtail** [comment]
if you're interested, I have a ready script:
https://github.com/RomkaSwallowtail/opencollar/blob/invert-descriptions/src/collar/oc_disgraced_themes.lsl

and description "converter":
https://github.com/RomkaSwallowtail/opencollar/blob/invert-descriptions/src/spares/Convert_Descriptions.lsl
